utilities/tag_point_locations.py

```diff
@@ -72,6 +77,32 @@
 )
 iem_gdf.to_crs(4326, inplace=True)
 
+# BBOX coordinates taken from an example aqi_forecast_24_hrs.tif file
+aqi_bbox = box(-1783505.405, 484131.432, 1134455.303, 2733401.487)
+aqi_gdf = gpd.GeoDataFrame({"geometry": [aqi_bbox]}, crs="EPSG:3338")
+
+
+def add_tags_within_polygon(communities, polygon_gdf, tag):
```

Comment:
Nice to see this function added - I could see this being used over and over to map the communities to their representative apps (and individual datasets within apps). Cool